src/main.cpp

```diff
@@ -1,18 +1,19 @@
 #include <Arduino.h>
 
-/**
- * Just implements a basic "blink" sketch.
- * on the esp32 dev board pin D2 controls the on-board LED.
- */
-#define LED_PIN IN_LED_PIN
+//
+#if defined ENABLE_LIGHTING
```

Comment:
i like this method of flagging what features are available, makes it easy to only include or exclude certain things.

we should probably do this for each major function